tests/src/environment/cnd_config.ts

```diff
@@ -71,9 +71,19 @@ interface Bitcoind {
     node_url: string;
 }
 
+interface BitcoinFees {
+    sat_per_vbyte?: number;
+    cypherblock?: {
+        // No point giving the option of "false" as absent == false
+        enabled: true;
+        blockchain_endpoint_url?: string;
+    };
+}
```

Comment:
What was the motivation for diverging from the design you used in nectar where the user chooses a 'strategy'?

Reply:
The fact that the user could use a "strategy" and then fill fields that are not used by the strategy, making it just confusing.

But yeah, maybe the best of both world is:
- use strategy
- fail to convert to `Settings` if any field NOT used by the selected strategy is set.

Reply:
I think I came up with a best design.